src/goto-symex/symex_function_call.cpp

```diff
@@ -288,7 +288,19 @@ void goto_symext::symex_function_call_post_clean(
 
   if(!goto_function.body_available())
   {
-    no_body(identifier);
+    // create a fatal assertion
+    if(symex_config.unwinding_assertions)
```

Comment:
Does this mean that this new behavior requires `--unwinding-assertions` to be passed to the command line explicitly ? Or is `unwinding_assertions` the default behaviour in v6.0 as well ?

Reply:
Yes, unwinding assertions are on by default in v6.